### ISSUE TITLE
protobuf enums support

### ISSUE THREAD

**Issue by Szpadel** [kondi/rxjs-grpc#1]
Hello,
Firstly, really great project!

Secondly, I found that below code generates RoleType as number value without defining enum.
It would be very helpful it that could be auto-generated.

Best Regards
Piotr

```syntax = "proto3";

package example;

service Service {
    rpc Call(Message) returns (Message) {}
}

message Message{
    required RoleType role = 1;
}

enum RoleType{
    ONE = 0;
    TWO = 1;
}
```

**Comment by kondi** [comment]
Cześć! :)

I am happy you like the project!

Thanks for the report. I have a partial fix now, the RoleType enum is generated, but the field type of the message is still a number. This is compatible with TypeScript, you can set and compare the enum value to a number.

```typescript
export interface Service {
  call(request: Message): Observable<Message>;
}

export interface Message {
  role: number;
}

export enum RoleType {
  ONE = 0,
  TWO = 1
}
```

However, you can realize this is not perfect solution when you try to set a value of a different enum type without getting a type error. The real fix needs more effort, I will do it in the following days.

Best Regards,
Tibor

**Comment by kondi** [comment]
Fix is included in version 0.1.0
